packages/wdio-config/src/lib/ConfigParser.ts

```diff
@@ -1,5 +1,11 @@
 import merge from 'deepmerge'
 import logger from '@wdio/logger'
+import { exists } from 'fs'
+import { promisify } from 'util'
+const fs = {
+    exists: promisify(exists)
+}
```

Comment:
Let's use:

```suggestion
import fs from 'fs/promises'
```

and do it this way:

```ts
const isExisting = await fs.access(pathToDir).then(() => true, () => false)
```